cmd/wallet-web/src/components/Vaults/VaultCard.vue

```diff
@@ -1,36 +1,42 @@
 <template>
-  <router-link
+  <div
     v-if="type === 'regular'"
-    :to="{ name: 'credentials', params: { vaultId } }"
     class="
       md:flex-col
-      px-6
-      xl:w-64
-      h-auto
-      md:h-40
       bg-neutrals-white
       rounded-xl
       border border-neutrals-dark border-opacity-20
       vaultContainer
       flex flex-row
+      relative
+      px-6
+      xl:w-64
+      h-auto
+      md:h-40
     "
   >
-    <div class="pt-5">
-      <div
-        :class="[`flex justify-center items-center w-12 h-12 bg-gradient-${color} rounded-full`]"
-      >
-        <img class="w-6 h-5" src="@/assets/img/vaults.svg" alt="Vault Icon" />
+    <router-link
+      :to="{ name: 'credentials', params: { vaultId } }"
+      class="flex flex-row md:flex-col"
+    >
+      <div class="pt-5">
+        <div
+          :class="[`flex justify-center items-center w-12 h-12 bg-gradient-${color} rounded-full`]"
+        >
+          <img class="w-6 h-5" src="@/assets/img/vaults.svg" alt="Vault Icon" />
+        </div>
```

Comment:
Why can't we just move `pt-5` to the inner `div` and get rid of the outer one? You have two `divs` wrapping up one `img`